CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 # Contributing
```

Comment:
I think I would prefer to either just delete this file or keep only the link to the development guide. It's easy to forget to update this.

Reply:
Sure